packages/expo-updates/src/Updates.ts

```diff
@@ -169,6 +170,42 @@ export async function checkForUpdateAsync(): Promise<UpdateCheckResult> {
   return result;
 }
 
+// @docsMissing
+/**
+ * Retrieves the most recent expo-updates log entries from the client, going back maxAge seconds
+ * (default: 3600 seconds = 1 hour)
+ *
+ * @return A promise that fulfills with an array of [`UpdatesLogEntry`](#updateslogentry) objects;
+ *
+ * The promise rejects if there is an unexpected error in retrieving the logs.
+ */
+export async function readLogEntriesAsync(maxAge?: number): Promise<UpdatesLogEntry[]> {
+  if (!ExpoUpdates.readLogEntriesAsync) {
+    throw new UnavailabilityError('Updates', 'readLogEntriesAsync');
+  }
+  const age = maxAge || 3600;
+  const result = await ExpoUpdates.readLogEntriesAsync(age);
```

Comment:
```suggestion
export async function readLogEntriesAsync(maxAge?: number = 3600): Promise<UpdatesLogEntry[]> {
  if (!ExpoUpdates.readLogEntriesAsync) {
    throw new UnavailabilityError('Updates', 'readLogEntriesAsync');
  }
  const result = await ExpoUpdates.readLogEntriesAsync(maxAge);
```

Simplify default value.

Reply:
Fixed